mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
I'm have a difficult time separating the declaration of unitSuffixes from the definition/initialization, mostly because it is entirely outside of any class, and because it involves strings.  I have tried a variety of things, including what I consider to be the ideal, which is declaring unitSuffixes this way:
<code>std::map&lt;Ms::Units, char*&gt;</code>
or like this:
<code>QMap&lt;Ms::Units, QString&gt;</code>
But without a class constructor I don't know where to initialize it.  And this doesn't compile:
<code>QString(QT_TRANSLATE_NOOP("context", "value");</code>
QString() doesn't like the curly braces inside the macro.
My lack of computer science education is exposing itself here.